source/production/F0.Minesweeper.Components/Minefield.razor

```diff
@@ -0,0 +1,23 @@
+<h3>Minefield</h3>
+
+@if (isValidSize)
+{
+	<table>
+		@for (uint currentHeight = 0; currentHeight < @Size.Height; currentHeight++)
```

Comment:
We have `Size.Height` in the outer loop, and `Size.Width` in the inner loop, but `Location` has `width`/`x` as first parameter and `height`/`y` as second parameter. To me it would be "more natural" the other way round.
(this suggestion goes in line with my commet on `MinefieldSize`)

Reply:
Also completely logical. I've exchanged the for loops to match your suggestion